### PR TITLE
Browser extensions sidebar update - Monetization article

### PR DESCRIPTION
**Pull request by rebloor**
Adding "Make money from browser extension" to the "Publishing at-arms" menu

### CODE REVIEW

**Comment by rebloor** [comment]
Thanks @wbamberg  I'm afraid I had assumed this was a new way of specifying the menus since I was last involved - presumably it's not possible to include comments in these files?.  I will shortly submit an update with the correct menu definition. In terms of adding the other locales, this article is being parked here before being moved to the new extensions workshop site, so I'm not sure whether adding information for localization here helps.

File: macros/AddonSidebar.ejs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<%
+If<%
```
Comment (wbamberg) [inline macros/AddonSidebar.ejs]: Not sure what this `If` is for.

File: macros/AddonSidebar.ejs [hunk 2]
```diff
@@ -408,12 +411,14 @@ var text = mdn.localStringMap({
       <details>
         <summary><%=text['Distribution/Publishing']%></summary>
         <ol>
+          <li><a href="<%=baseURL%>Distribution"><%=text['Distribution/Signing']%></a></li>
```
Comment (wbamberg) [inline macros/AddonSidebar.ejs]: I think this is not intended? It just seems to duplicate the previous item.

File: macros/AddonSidebar.ejs [hunk 3]
```diff
@@ -408,12 +411,14 @@ var text = mdn.localStringMap({
...
           <li><a href="<%=baseURL%>Distribution"><%=text['Distribution/Signing']%></a></li>
           <li><a href="<%=baseURL%>WebExtensions/Package_your_extension_"><%=text['WebExtensions/Package_your_extension_']%></a></li>
           <li><a href="<%=baseURL%>Distribution/Submitting_an_add-on"><%=text['Distribution/Submitting_an_add-on']%></a></li>
           <li><a href="<%=baseURL%>Source_Code_Submission"><%=text['Source_Code_Submission']%></a></li>
           <li><a href="<%=baseURL%>Distribution/Resources_for_publishers"><%=text['Distribution/Resources_for_publishers']%></a></li>
           <li><a href="<%=baseURL%>Listing"><%=text['Listing']%></a></li>
+          <li><a href="<%=baseURL%>Distribution/Make_money_from_browser_extensions"><%=text['Distribution/Make_money_from_browser_extensions']%>
```
Comment (wbamberg) [inline macros/AddonSidebar.ejs]: This should close the `<a>` and `<li>` tags

File: macros/AddonSidebar.ejs (hunk 1 above)
Comment (rebloor) [reply macros/AddonSidebar.ejs]: Story, must've been a stray copy paste, removed in latest commit

File: macros/AddonSidebar.ejs (hunk 2 above)
Comment (rebloor) [reply macros/AddonSidebar.ejs]: Removed in latest commit

File: macros/AddonSidebar.ejs (hunk 3 above)
Comment (rebloor) [reply macros/AddonSidebar.ejs]: fixed in the latest commit

**Pull request merged**